Makefile

```diff
@@ -172,7 +230,9 @@ run-contract:
 	make project-stop
 
 run-smoke:
+	sed -i -e 's|AUTH_REPLACE|$(AUTHENTICATION_ENDPOINT)|g' $(APPLICATION_TEST_DIR)/contract/environment/postcode_smoke.postman_environment.json
 	sed -i -e 's|SECRET_TO_REPLACE|$(COGNITO_USER_PASS)|g' $(APPLICATION_TEST_DIR)/contract/environment/postcode_smoke.postman_environment.json
```

Comment:
Do we still need AUTH_REPLACE and SECRET_TO_REPLACE since we are no longer connecting to the authentication endpoint for the smoke tests?